bindings/go/src/fdb/range.go

```diff
@@ -249,40 +248,7 @@ func (ri *RangeIterator) fetchNextBatch() {
 	ri.iteration++
 
 	f := ri.t.doGetRange(ri.sr, ri.options, ri.snapshot, ri.iteration)
-	ri.f = &f
-}
-
-// Get returns the next KeyValue in a range read, or an error if one of the
-// asynchronous operations associated with this range did not successfully
-// complete. The Advance method of this RangeIterator must have returned true
-// prior to calling Get.
-func (ri *RangeIterator) Get() (kv KeyValue, err error) {
-	if ri.err != nil {
-		err = ri.err
-		return
-	}
-
-	kv = ri.kvs[ri.index]
```

Comment:
Note: on next advance call, this slice is thrown away. I'd like to address this in this PR, or a separate one, by having `Advance()` return the whole slice.

Reply:
I ended up switching from `Advance()`/`GetSlice()` to `NextBatch()`/`Get()`